compute/etc/pgbouncer.ini

```diff
@@ -21,6 +21,8 @@ unix_socket_dir=/tmp/
 unix_socket_mode=0777
 ; required for pgbouncer_exporter
 ignore_startup_parameters=extra_float_digits
+; pidfile for graceful termination
+pidfile=/etc/pgbouncer/pid
```

Comment:
`/etc` is not a good location for PID files. Please use either the `XDG_RUNTIME_DIR` or `/tmp`. Getting access to `XDG_RUNTIME_DIR` is not going to be possible, so `/etc/pgbouncer.pid` would be good.

Reply:
I see that local_proxy puts its pid file into `/etc`. Let's not follow that pattern.

Reply:
https://github.com/neondatabase/neon/pull/8823#discussion_r1780335243 :D

Reply:
At least I'm consistent :laughing:. `/tmp` it is!